<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogCreateProject</class>
 <widget class="QDialog" name="DialogCreateProject">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>130</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="projectType">
     <property name="leftMargin">
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="projectTypeLabel">
       <property name="text">
        <string>Project type:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="projectTypeValue">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="projectName">
     <property name="leftMargin">
      <number>4</number>
     </property>
     <item>
      <widget class="QLabel" name="projectNameLabel">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Project name:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="projectNameValue"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="projectLocation">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="projectLocationLabel">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Project path:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="projectLocationPath">
       <property name="spacing">
        <number>4</number>
       </property>
       <item>
        <widget class="QToolButton" name="projectLocationChoose">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="projectLocationValue"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="projectDialog">
     <item>
      <widget class="QPushButton" name="projectDialogCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="projectDialogCreate">
       <property name="text">
        <string>Create</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
